tetrateci/create_istio_release.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+mkdir /tmp/istio-release
+sudo gem install fpm
```

Comment:
Lets not install any extra packages explicitly stated in the release builder. We shall use the container based build so that all the dependencies are always covered.

Reply:
Build container image needs to be picked from https://github.com/istio/test-infra/blob/master/prow/config/jobs/release-builder.yaml and set through env var IMAGE_VERSION.

Reply:
We dont have  a reliable way to set the image version, what do you suggest?